homeassistant/components/telegram_bot/webhooks.py

```diff
@@ -127,7 +142,10 @@ async def stop_application(self, event=None):
     async def deregister_webhook(self):
         """Query telegram and deregister the URL for our webhook."""
         _LOGGER.debug("Deregistering webhook URL")
-        await self.bot.delete_webhook()
+        try:
+            await self.bot.delete_webhook()
+        except:  # noqa: E722
```

Comment:
We should always name the exception that we want to catch. Check the library docs for what exception can be raised here.